dexec/cmd.go

```diff
@@ -120,35 +120,44 @@ func (c *Cmd) logiofn(prefix string) func(string) {
 		if c.Process != nil {
 			pid = c.Process.Pid
 		}
-		dlog.Printf(c.ctx, "[pid:%v] %s %s", pid, prefix, msg)
+		ctx := dlog.WithField(c.ctx, "dexec.pid", pid)
+		ctx = dlog.WithField(ctx, "dexec.stream", stream)
+		if msg != nil {
+			ctx = dlog.WithField(ctx, "dexec.data", string(msg))
+		}
+		if err != nil {
+			ctx = dlog.WithField(ctx, "dexec.err", err)
+		}
+		dlog.Print(ctx)
```

Comment:
this looks a little confusing until you realize that it's just logging the fields set above (which i figured out from the diff) -- a short comment explaining what is being logged might be good here

Reply:
I've added a comment.